ehrapy/io/_utility_io.py

```diff
@@ -28,3 +28,31 @@ def _get_file_extension(file_path: Path) -> str:
         {supported_extensions}
         """
     )
+
+
+def _check_columns_only_params(
+    obs_only: dict[str, list[str]] | list[str] | None, x_only: dict[str, list[str]] | list[str] | None
+) -> None:
+    """Check whether columns_obs_only and columns_x_only can be used as passed. For a single anndata object (thus
+    parameters being a list of strings) it's not possible to pass both, obs_only and x_only.
+    For multiple anndata objects (thus the parameters being dicts of string keys with a list value), it is possible to pass both. But the keys
+    (unique identifiers of the anndata objects, basically its names) should share no common identifier, thus a single anndata object is either in x_only OR
+    obs_only, but not in both.
+    """
+    # at least one parameter is None
```

Comment:
Well, that's what the code says. Useless comment.
Rather write something like: "At least one of A and B is None. Simply return because X Y Z"